pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -121,3 +135,17 @@ func removeEmptyStrings(s []string) []string {
 	}
 	return nonempty
 }
+
+//GenerateSecureString creates a webhook secret for you.
+func GenerateSecureString() (string, error) {
```

Comment:
You should add some tests for this baby.

Reply:
I will make that change

Reply:
Haha..!!Test case added.